aspnetcore/performance/response-compression.md

```diff
@@ -125,11 +125,14 @@ The following code:
 
 Set the compression level with <xref:Microsoft.AspNetCore.ResponseCompression.BrotliCompressionProviderOptions> and <xref:Microsoft.AspNetCore.ResponseCompression.GzipCompressionProviderOptions>. The Brotli and Gzip compression providers default to the fastest compression level, [CompressionLevel.Fastest](xref:System.IO.Compression.CompressionLevel), which might not produce the most efficient compression. If the most efficient compression is desired, configure the response compression middleware for optimal compression.
 
-| Compression Level | Description |
-| ----------------- | ----------- |
-| [CompressionLevel.Fastest](xref:System.IO.Compression.CompressionLevel) | Compression should complete as quickly as possible, even if the resulting output isn't optimally compressed. |
-| [CompressionLevel.NoCompression](xref:System.IO.Compression.CompressionLevel) | No compression should be performed. |
-| [CompressionLevel.Optimal](xref:System.IO.Compression.CompressionLevel) | Responses should be optimally compressed, even if the compression takes more time to complete. |
+| Compression Level | Description | Gzip level | Brotli level |
+| ----------------- | ----------- | ---------- | ------------ |
+| [CompressionLevel.Fastest](xref:System.IO.Compression.CompressionLevel) | Compression should complete as quickly as possible, even if the resulting output isn't optimally compressed. | 1 | 1 |
+| [CompressionLevel.NoCompression](xref:System.IO.Compression.CompressionLevel) | No compression should be performed. | 0 | 0 |
+| [CompressionLevel.Optimal](xref:System.IO.Compression.CompressionLevel) | Compression should optimally balance compression speed and output size. | -1 (6) | 4 |
```

Comment:
Is -1 a valid Gzip level? What does 6 in parentheses mean?

Reply:
Why duplicate all that content here and play catch-up. Why not

See [CompressionLevel Enum](https://review.learn.microsoft.com/en-us/dotnet/api/system.io.compression.compressionlevel) for values that indicate whether a compression operation emphasizes speed or compression size.

Reply:
> Why duplicate all that content here and play catch-up. Why not
> 
> See [CompressionLevel Enum](https://review.learn.microsoft.com/en-us/dotnet/api/system.io.compression.compressionlevel) for values that indicate whether a compression operation emphasizes speed or compression size.

I agree that it would be best to replace the table with this sentence, and omit the Brotli and Gzip numbers as implementation details per @gewarren's suggestion.